synths/core-synths.scd

```diff
@@ -124,14 +124,14 @@ live coding them requires that you have your SuperDirt instance in an environmen
 
 	SynthDef("dirt_hpf" ++ numChannels, { |out, hcutoff = 440, hresonance = 0|
 		var signal = In.ar(out, numChannels);
-		signal = RHPF.ar(signal, hcutoff.abs, hresonance.linexp(0, 1, 1, 0.001));
+		signal = RHPF.ar(signal, hcutoff.abs.clip(20,80e3), hresonance.linexp(0, 1, 1, 0.001));
```

Comment:
just for curiosity: how have you found that magic number?

Reply:
No magic, just late at night and I forgot about SampleRate.ir (I've updated the PR)

Some people are particular about high frequencies so I was reluctant to just chop at 20 kHz.  But such people would be using a high server sampling rate, so I think it's sensible to just filter at the Nyquist frequency.